packages/apollos-church-api/src/data/rock-constants/index.js

```diff
@@ -0,0 +1,52 @@
+import { RockConstants as apollosRockConstants } from '@apollosproject/data-connector-rock';
+import ApollosConfig from '@apollosproject/config';
+
+const { ROCK_MAPPINGS } = ApollosConfig;
+
+const mapNewSpringNameToRockName = (name) => {
```

Comment:
Why is this function even needed? All it does is say if 'PrayerRequest' equals 'PrayerRequest'... Which it always would.

Reply:
Resolved.